internal/service/ipam/api_address_block_resource_test.go

```diff
@@ -221,6 +221,38 @@ func TestAccAddressBlockResource_Comment(t *testing.T) {
 	})
 }
 
+// The provider does not support creating user roles , which is a required field for creating a compartment
```

Comment:
you can set up the proper roles manually in the account we use to run acceptance tests.

Reply:
Will add this in a separate PR .